help/destinations/catalog/crm/outreach.md

```diff
@@ -0,0 +1,208 @@
+*  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+
+### Enable alerts {#enable-alerts}
+
+You can enable alerts to receive notifications on the status of the dataflow to your destination. Select an alert from the list to subscribe to receive notifications on the status of your dataflow. For more information on alerts, see the guide on [subscribing to destinations alerts using the UI](../../ui/alerts.md).
+
+When you are finished providing details for your destination connection, select **[!UICONTROL Next]**.
+
+## Activate segments to this destination {#activate}
+
+> [!IMPORTANT]
+> 
+> To activate data, you need the **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+Read [Activate profiles and segments to streaming segment export destinations](../../ui/activate/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination.
+
+### Mapping considerations and example {#mapping-considerations-example}
+
+To correctly send your audience data from Adobe Experience Platform to the Outreach destination, you need to go through the field mapping step. Mapping consists of creating a link between your Experience Data Model (XDM) schema fields in your Platform account and their corresponding equivalents from the target destination. To correctly map your XDM fields to the Outreach destination fields, follow these steps:
+
+1. In the Mapping step, click **[!UICONTROL Add new mapping]**, you will see a new mapping row on the screen.
+![Add new mapping](../../assets/catalog/crm/outreach/add-new-mapping.png)
+
+1. In the select source field window, when selecting the source field choose the **[!UICONTROL Select identity namespace]** category and add the mappings desired.
+![Source mapping](../../assets/catalog/crm/outreach/source-mapping.png)
+
+1. In the select target field window, select the target field and choose the **[!UICONTROL Select identity namespace]** category and add the mappings desired.
+![Target mapping using contactId](../../assets/catalog/crm/outreach/target-mapping.png)
+
+1. For custom attributes, in the select target field window, select the target field and choose the **[!UICONTROL Select custom attributes]** category, Next provide the desired target attribute name and add the mappings desired.
+![Target mapping using LastName](../../assets/catalog/crm/outreach/target-mapping-lastname.png)
+
+1. For instance, you could add the following mapping between your XDM profile schema and your [!DNL Outreach] instance:
+
+    ||XDM Profile Schema|[!DNL Outreach] Instance| Mandatory|
```

Comment:
What should the user understand from the Mandatory column? Is the Outreach ID identity mandatory mapping for everyone? If it is, we should call this out in the mapping step.  Also, the Attributes example doesn't have any mandatory column value. Should it be Yes/No?

Reply:
_What should the user understand from the Mandatory column? Is the Outreach ID identity mandatory mapping for everyone? If it is, we should call this out in the mapping step._ 
Added additional notes as suggested,

_Also, the Attributes example doesn't have any mandatory column value. Should it be Yes/No?_
It depends on the attributes that the end user wants to update. Hence there is no input provided. Have added verbiage, trust it would work.